next.config.js

```diff
@@ -4,6 +4,7 @@ const nextConfig = {
   swcMinify: true,
   basePath: process.env.NEXT_PUBLIC_BASE_PATH,
   staticPageGenerationTimeout: 900, // equivalent to netlify's timeout
+  transpilePackages: ['@storybook/components-marketing'],
```

Comment:
@yannbf — This is to fix the issue with `@storybook/api/shortcut`.

- That package is used by `@storybook/components-marketing`, to display the keyboard shortcut for the search in the header.
- `@storybook/components-marketing` has a dependency on `"@storybook/api": "^6.5.9"`, so it's not a matter of the `api` package being removed in SB 7
- It's instead a transpilation error, thus this new config (and the upgrade to next 13.1 to be able to use it)

Reply:
This issue propagated to Storybook as well, I fixed it by patching the @storybook/components-marketing package. Instead of importing from `@storybook/api/shortcut`, importing from `@storybook/manager-api` instead.

I think it would make more sense if `@storybook/components-marketing` stopped depending on `@storybook/api` and just copied those minimal functions to use internally